ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-gas-limit/advanced-gas-fee-gas-limit.js

```diff
@@ -58,9 +58,10 @@ const AdvancedGasFeeGasLimit = () => {
   }
 
   return (
-    <Typography
-      tag={TypographyVariant.paragraph}
-      variant={TypographyVariant.H7}
+    <Text
+      tag={TextVariant.bodyMd}
```

Comment:
I believe tag for the Typography component is the `as` prop for the Text component so we can remove this
```suggestion
```